### DISCUSSION TITLE
Integrate with k6

### DISCUSSION THREAD

**Discussion by george4joseph** [apideck-libraries/portman#276] [Q&A] [answered]
Hi,
It will a great work if we can integrate portman with k6 [https://k6.io/](url).
Possible way of approach `Converting Openapi.json to k6 script`

**Comment by thim81** [comment, marked as answer]
hi @george4joseph 

We gave a talk about the option to use Portman with K6 and maintain a NPM package [postman-to-k6](https://github.com/apideck-libraries/postman-to-k6)
https://www.youtube.com/watch?v=Be66Db4wHLA

It explains how we use OpenAPI -> Postman (via Portman) -> K6

![image](https://user-images.githubusercontent.com/952446/157851271-b2ba3ca9-cb6f-4717-ba42-f2c3af8193d2.png)

![image](https://user-images.githubusercontent.com/952446/157851250-6a5f4bcb-02d4-4973-9e64-89299fb5c40d.png)
